docs/math/maxwell.md

```diff
@@ -1,70 +1,147 @@
+    \mathbf{\nabla}\times\mathbf{H}\left(\mathbf{r},t\right)&= \mathbf{J}_{ext}\left(\mathbf{r},t\right) + \frac{\partial \mathbf{D}\left(\mathbf{r},t\right)} {\partial t} 
+\end{aligned}$$ 
 
-    &\vec{\mathcal{D}} = \epsilon_0 \epsilon_r \vec{\mathcal{E}}
+with the four fields (note that a bold letter indicates a vector)
 
-    &\vec{\mathcal{B}} = \mu_0 \mu_r \vec{\mathcal{H}}
-$$
+-   $\mathbf{D}\left(\mathbf{r},t\right)$: electric displacement
 
-$$
-    &\omega = 2 \pi f
+-   $\mathbf{E}\left(\mathbf{r},t\right)$: electric field
 
-    &\lambda = \frac{c}{f} = \frac{2 \pi c}{\omega} = \frac{2 \pi}{k}
+-   $\mathbf{H}\left(\mathbf{r},t\right)$: magnetic displacement
 
-    &k = \frac{2 \pi}{\lambda} = \frac{2 \pi f}{c} = \frac{\omega}{c}
-$$
+-   $\mathbf{B}\left(\mathbf{r},t\right)$: magnetic field
 
-## Eigenvectors propagating in $x_3$-direction
+In the equations $\varrho_{\text{ext}}$ and $\mathbf{J}_{\text{ext}}$ are the external charge and current density. These are macroscopic equations, i.e., local average over microscopic quantities.
 
-Assuming no sources and currents present, {eq}`maxwell` simplifies to
+The coupling to matter can be described in the macroscopic equations via the polarisation $\mathbf{P}$ and the magnetisation $\mathbf{M}$
 
-$$
-    &\nabla\cdot \left(\varepsilon\vec{\mathcal{E}}\right) = 0
+$$\begin{aligned}
+    \mathbf{D}\left(\mathbf{r},t\right) &=& \varepsilon_0 \mathbf{E}\left(\mathbf{r},t\right) + \mathbf{P}\left(\mathbf{r},t\right)\\
+    \mathbf{H}\left(\mathbf{r},t\right) &=& \frac{1}{\mu_0} \mathbf{B}\left(\mathbf{r},t\right)- \mathbf{M}\left(\mathbf{r},t\right)    
+\end{aligned}$$ 
 
-    &\nabla\cdot \left(\mu\vec{\mathcal{H}}\right) = 0
+with $\varepsilon_0$ the electric permittivity and $\mu_0$ the magnetic permeability. The polarisation $\mathbf{P}$ describes the dipole moment per unit cell of the material
 
-    &
-    \nabla\times\vec{\mathcal{E}}
-    =
-    - \mu \frac{\partial \vec{\mathcal{H}}}{\partial t}
+$$\begin{aligned}
+    \mathbf{\nabla}\cdot\mathbf{P}\left(\mathbf{r},t\right) = - \varrho_{\text{int}}\left(\mathbf{r},t\right) \notag
+\end{aligned}$$ 
 
-    &
-    \nabla\times\vec{\mathcal{H}}
-    =
-    \varepsilon\frac{\partial \vec{\mathcal{E}}}{\partial t}
-$$ (maxwell_no_sources)
+Now putting these back in the Maxwell equations we obtain 
+
+$$\begin{aligned}
+    \mathbf{\nabla}\cdot\mathbf{D}\left(\mathbf{r},t\right)=& \mathbf{\nabla}\cdot\left(\varepsilon_0 \mathbf{E}\left(\mathbf{r},t\right) + \mathbf{P}\left(\mathbf{r},t\right) \right) \\
+        \Rightarrow 
+        \mathbf{\nabla}\cdot\mathbf{E}\left(\mathbf{r},t\right) =& \frac{1}{\varepsilon_0} \left(  \mathbf{\nabla}\cdot\mathbf{D}\left(\mathbf{r},t\right) -  \mathbf{\nabla}\cdot\mathbf{P}\left(\mathbf{r},t\right) \right)
+        = \frac{1}{\varepsilon_0} \left( \rho_{\text{ext}}\left(\mathbf{r},t\right)+\rho_{\text{int}}\left(\mathbf{r},t\right) \right)\varrho\,
```

Comment:
```suggestion
        = \frac{1}{\varepsilon_0} \left( \rho_{\text{ext}}\left(\mathbf{r},t\right)+\rho_{\text{int}}\left(\mathbf{r},t\right) \right)\,
```

The $\varrho$ shouldn't be there, right?

Reply:
true and the comma shouldn't be there, too